src/JarvisDesk.cpp

```diff
@@ -242,9 +268,11 @@ class JarvisDesk_impl {
       Log.println(h);
       return;
     }
-
     h = Util::to_mm(h);
-    if (height == h) return;
+    if (height == h || h < 640 || h > 1300)
```

Comment:
Why are these out of range?  Can you math here or use a #define?

Reply:
This was the range of quite of the few desks I found. I will add them to a #define in the pinout config.

Reply:
`to_mm()` already does `h = height * 2.54` when `height < 600`.  So `h > 1300` should only happen if `height > 1300` as well, I think.  I already filtered out "9999" as a bogus value I receive from my desk sometimes.  I wonder what other sentinel values we could see here.